### PR TITLE
Split tests into one workflow for each framework.

### PR DESCRIPTION
**Pull request by jaheba**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

**Please tag this pr with at least one of these labels to make our release process faster:** BREAKING, new feature, bug fix, other change, dev setup

### CODE REVIEW

File: src/gluonts/mx/distribution/empirical_distribution.py [hunk 1]
```diff
@@ -86,23 +86,19 @@ def stddev(self) -> Tensor:
...
-        self.CDF_sorted = self.F.linspace(
-            start=1 / self.samples.shape[0],
-            stop=1,
-            endpoint=True,
-            num=self.samples.shape[0],
-        )
 
-        # Replace this when mxnet is updated to a version where `searchsorted` is available!
-        import torch
+        ix = [
```
Comment (mbohlkeschneider) [inline src/gluonts/mx/distribution/empirical_distribution.py:90]: Isn't there an `mx.searchsorted` at this point?

Comment (jaheba) [reply src/gluonts/mx/distribution/empirical_distribution.py:90]: There is `mx.np.searchsorted`, which AFAIK means that you should be able to do `np.searchsorted` on `mx.nd.array`.

HOWEVER, the behaviour is different and it doesn't work.

Comment (jaheba) [reply src/gluonts/mx/distribution/empirical_distribution.py:90]: Also, `torch` behaves different in the sense that you can use 2d arrays, whilst numpy only supports 1d.

Comment (mbohlkeschneider) [reply src/gluonts/mx/distribution/empirical_distribution.py:90]: I see. Too bad.

**Review by mbohlkeschneider — APPROVED** [review]
